### PR TITLE
Lokalise: Translations update

### PR DESCRIPTION
**Pull request by jyclam**
This pull request was initiated by Lokalise (user Agnieszka Galeza) at 2020-11-10 22:31:13

### CODE REVIEW

File: client/src/locales/translations/en_GB.json [hunk 1]
```diff
@@ -267,7 +267,7 @@
       "myActivity": "My Activity",
       "userActivity": "User Activity",
       "wantHelp": "I want to help",
-      "needHelp": "I need help",
+      "needHelp": "I need help ",
```
Comment (Zaydme) [inline client/src/locales/translations/en_GB.json:270]: A trailing space here but I think that's fine

Comment (galeza) [reply client/src/locales/translations/en_GB.json:270]: I saw this trailing space in Eng-US.

File: client/src/locales/translations/pl.json [hunk 2]
```diff
@@ -363,14 +363,39 @@
...
     "ourCommunity": "Nasza społeczność",
-    "community": "Nasz zespół składa się z ponad 450 wolontariuszy: programistów, projektantów, naukowców, ekspertów w dziedzinie zdrowia i menedżerów produktów z całego świata",
+    "community": "Nasz zespół składa się z ponad 1100 wolontariuszy: programistów, projektantów, naukowców, ekspertów w dziedzinie zdrowia i menedżerów produktów z całego świata",
     "ourIdea": "Nasz pomysł",
     "ourSupporters": "Wspierają nas:",
     "followUs": "Śledź nas w mediach społecznościowych",
-    "contactUs": "Lub skontaktuj się z nami pod adresem",
-    "contactEmail": "contact@fightpandemics.com"
+    "contactUs": "Lub skontaktuj się z nami pod adresem: ",
+    "contactEmail": "contact@fightpandemics.com",
```
Comment (rabbi08) [inline client/src/locales/translations/pl.json:373]: contactEmail: admin@fightpandemics.com

File: client/src/locales/translations/en_GB.json (hunk 1 above)
Comment (Zaydme) [reply client/src/locales/translations/en_GB.json:270]: Then maybe it's supposed to be there.

**Review by Zaydme — APPROVED** [review]

File: client/src/locales/translations/pl.json (hunk 2 above)
Comment (galeza) [reply client/src/locales/translations/pl.json:373]: @rabbi08 
I'll trigger PR once more

**Pull request closed without merging**